src/operator/src/insert.rs

```diff
@@ -676,17 +676,11 @@ impl Inserter {
 
         match res {
             Ok(table) => {
-                info!(
-                    "Successfully created a log table {}.{}.{}",
-                    table_ref.catalog, table_ref.schema, table_ref.table,
-                );
+                info!("Successfully created a log table {}", table_ref);
                 Ok(table)
             }
             Err(err) => {
-                error!(
-                    "Failed to create a log table {}.{}.{}: {}",
-                    table_ref.catalog, table_ref.schema, table_ref.table, err
-                );
+                error!(err; "Failed to create a log table {}", table_ref);
```

Comment:
Could you please also fix other similar error logging issues in this mod?
https://github.com/GreptimeTeam/greptimedb/blob/11a903f19313e27bc0efb5025557f43c399fef06/src/operator/src/insert.rs#L578-L581

Reply:
I did a search in this mod. Didn't find anything else like it.

Reply:
My bad. I found some one. I will fix it asap.